mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -961,10 +960,28 @@ message GetNetworkStatusResponse {
     // Whether we are behind.
     bool is_behind = 4;
 
-    // The latest block info reported by a consensus node
-    consensus_common.LastBlockInfoResponse last_block_info = 5;
+    // The latest block info data reported by a consensus node
+    LastBlockInfo last_block_info = 5;
 }
 
+// Data about the network state and last block processed by the consensus network
+message LastBlockInfo {
```

Comment:
:question: Double checking my understanding from this comment
> I think this change is good from a separation of concerns point of view anyway. Consensus should be able to evolve its API without worrying if it's breaking mobilecoind users.

The `LastBlockInfo` was being reused from consensus due to convenience and not because they needed to stay n'sync?
This suggests there is little to no benefit to trying to place the `LastBlockInfo` in a more common proto like `blockchain`.

Reply:
yeah i think that's right, it was just a pass through, but i wasn't sure if that was a good idea at the time.